.circleci/config.yml

```diff
@@ -303,7 +321,7 @@ jobs:
       - <<: *jdk_8
 
 workflows:
-  version: 2
+  version: 3
```

Comment:
Shouldn't it be 2? https://circleci.com/docs/2.0/configuration-reference/#version-1

It works though.

Reply:
You're right. Fixed now.